### ISSUE TITLE
msvc compile error

### ISSUE THREAD

**Issue by tbraun96** [rust-num/num-bigint#224]
```
C:\satori>cargo build --package num-bigint:0.4.0
   Compiling libm v0.2.1
   Compiling num-traits v0.2.14
   Compiling num-integer v0.1.44
   Compiling num-bigint v0.3.2
error[E0658]: use of unstable library feature 'int_roundings'
  --> C:\Users\tbrau\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.3.2\src\biguint\convert.rs:70:10
   |
70 |         .div_ceil(&big_digit::BITS.into())
   |          ^^^^^^^^
   |
   = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
   = help: add `#![feature(int_roundings)]` to the crate attributes to enable

error[E0308]: mismatched types
  --> C:\Users\tbrau\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.3.2\src\biguint\convert.rs:70:19
   |
70 |         .div_ceil(&big_digit::BITS.into())
   |                   ^^^^^^^^^^^^^^^^^^^^^^^ expected `u64`, found reference
   |
   = note:   expected type `u64`
           found reference `&_`
help: consider removing the borrow
   |
70 -         .div_ceil(&big_digit::BITS.into())
70 +         .div_ceil(big_digit::BITS.into())
   |

error[E0658]: use of unstable library feature 'int_roundings'
   --> C:\Users\tbrau\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.3.2\src\biguint\convert.rs:585:10
    |
585 |         .div_ceil(&u64::from(bits))
    |          ^^^^^^^^
    |
    = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
    = help: add `#![feature(int_roundings)]` to the crate attributes to enable

error[E0308]: mismatched types
   --> C:\Users\tbrau\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.3.2\src\biguint\convert.rs:585:19
    |
585 |         .div_ceil(&u64::from(bits))
    |                   ^^^^^^^^^^^^^^^^ expected `u64`, found `&u64`
    |
help: consider removing the borrow
    |
585 -         .div_ceil(&u64::from(bits))
585 +         .div_ceil(u64::from(bits))
    |

error[E0658]: use of unstable library feature 'int_roundings'
   --> C:\Users\tbrau\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.3.2\src\biguint\convert.rs:613:10
    |
613 |         .div_ceil(&u64::from(bits))
    |          ^^^^^^^^
    |
    = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
    = help: add `#![feature(int_roundings)]` to the crate attributes to enable

error[E0308]: mismatched types
   --> C:\Users\tbrau\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.3.2\src\biguint\convert.rs:613:19
    |
613 |         .div_ceil(&u64::from(bits))
    |                   ^^^^^^^^^^^^^^^^ expected `u64`, found `&u64`
    |
help: consider removing the borrow
    |
613 -         .div_ceil(&u64::from(bits))
613 +         .div_ceil(u64::from(bits))
    |

error[E0658]: use of unstable library feature 'int_roundings'
   --> C:\Users\tbrau\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.3.2\src\biguint.rs:393:45
    |
393 |                 let root_scale = extra_bits.div_ceil(&n64);
    |                                             ^^^^^^^^
    |
    = note: see issue #88581 <https://github.com/rust-lang/rust/issues/88581> for more information
    = help: add `#![feature(int_roundings)]` to the crate attributes to enable

error[E0308]: mismatched types
   --> C:\Users\tbrau\.cargo\registry\src\github.com-1ecc6299db9ec823\num-bigint-0.3.2\src\biguint.rs:393:54
    |
393 |                 let root_scale = extra_bits.div_ceil(&n64);
    |                                                      ^^^^ expected `u64`, found `&u64`
    |
help: consider removing the borrow
    |
393 -                 let root_scale = extra_bits.div_ceil(&n64);
393 +                 let root_scale = extra_bits.div_ceil(n64);
    |

Some errors have detailed explanations: E0308, E0658.
For more information about an error, try `rustc --explain E0308`.
error: could not compile `num-bigint` due to 8 previous errors
```

**Comment by cuviper** [comment]
> ```
> C:\satori>cargo build --package num-bigint:0.4.0
>    Compiling libm v0.2.1
>    Compiling num-traits v0.2.14
>    Compiling num-integer v0.1.44
>    Compiling num-bigint v0.3.2
> ```

Strange that you explicitly gave it version 0.4.0, yet it's building 0.3.2. That may be a cargo issue of its own...

Anyway, this is a known conflict with the new unstable methods, discussed in rust-lang/rust#88581. That might yet be changed or reverted, but still I have published workarounds in `num-bigint` 0.3.3 and 0.4.2, depending on which semver you need. You should be able to `cargo update -p num-bigint` to get the new versions.

**Comment by tbraun96** [comment]
Perfect, thanks
